dotnet-algorand-sdk/V2/Indexer/Model/Model.cs

```diff
@@ -783,7 +783,7 @@ public StateSchema(ulong? numUint = 0, ulong? numByteSlice = 0)
             // to ensure "numByteSlice" is required (not null)
             if (numByteSlice == null)
             {
-                throw new InvalidDataException("numByteSlice is a required property for StateSchema and cannot be null");
+                //throw new InvalidDataException("numByteSlice is a required property for StateSchema and cannot be null");
```

Comment:
This conditional is janky. Is there a reason to keep it that I'm not seeing?

Reply:
Auto-generated. The JSON spec says that the fields are required:

`  "StateSchema": {
      "description": "Represents a \\[apls\\] local-state or \\[apgs\\] global-state schema. These schemas determine how much storage may be used in a local-state or global-state for an application. The more space used, the larger minimum balance must be maintained in the account holding the data.",
      "type": "object",
      "required": [
        "num-uint",
        "num-byte-slice"
      ],`

Reply:
yeah, the schema might be wrong.. the reality is that it is failing parsing data

Reply:
I have found the go models and StateSchema properties should be non nullable but required.